### PR TITLE
doc reorg: created concepts/security/features and move existing pages

### PR DESCRIPTION
**Pull request by RemyDeWolf**
Summary of changes

- Break down the existing ***Tutorial*** pages into a few buckets:
    - ***Features***: Each section describes a high level feature = what it does + how to use it. A feature could have multiple pages if there is a lot to cover.
        - ex: Secrets - 1 page
        - ex: Identity and Access Management (IAM) - 3 pages
    - ***References/cloud***: cloud specific documentation
        - ex: EKS Update
- Some pages would be renamed
    - old name → new name. Mostly to use shorter titles and be consistent with other pages.
- **to review content** -> Create a ***Concepts*** section to explain the abstractions used by opta
- compliance & architecture/security are now all under **Security**

Related changes to opta repo: https://github.com/run-x/opta/pull/742

https://www.loom.com/share/b904c02da77f40b1ab94c22d5b38f78b

### CODE REVIEW

File: content/en/Concepts/environment.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+linkTitle: "Environment"
+draft: false
+weight: 2
+description: >
+  The common frame that powers your infrastructure.
+---
+
+## What is an Environment?
+
+An Environment is the common frame that powers your infrastructure.
```
Comment (NitinAgg) [inline content/en/Concepts/environment.md:12]: This is too abstract. Checkout the definition which is here: https://github.com/run-x/opta-docs/pull/150

File: content/en/Concepts/layer.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    links:
+      - mongodb:
+         -  mongodb_atlas_connection_string: MONGODB_URI
+            db_user: MONGODB_USER
+            db_password: MONGODB_PASSWORD
+  - name: mongodb
+    type: mongodb-atlas
+```
+
+## When to use layers?
```
Comment (NitinAgg) [inline content/en/Concepts/layer.md:45]: Let's move this section up. Just after the definition

File: content/en/Concepts/overview.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Opta is based around the concept of Infrastructure-As-Code. You write configuration files and then run the Opta CLI -
+which connects to your cloud account and sets things up to reflect the configuration. The Opta CLI can be run from your local machine or a CI/CD system (like jenkins or Github actions).
+
+There are two primary kinds of configuration files:
+
+- **Environment**: This file specifies which cloud/account/region should Opta be set up in. Running Opta on this creates all the
+  base resources like a kubernetes cluster, networks, IAM roles, ingress, service mesh, etc. Usually, you'd have 1 env for staging, 1 for
+  prod, 1 for qa, etc. You can also do one environment per engineer or pull request - which gives everyone an isolated sandbox to play in!
+- **Template**: This file specifies the workload you want to run (usually a microservice). You can also specify any non-k8s resources
```
Comment (NitinAgg) [inline content/en/Concepts/overview.md:20]: Why template and not layer or service?

**Review by NitinAgg — COMMENTED** [review]
This is great!

File: content/en/Concepts/overview.md (hunk 3 above)
Comment (RemyDeWolf) [reply content/en/Concepts/overview.md]: updated to layer to match the new page, good catch

**Comment by RemyDeWolf** [comment]
@NitinAgg addressed the comments

**Review by NitinAgg — APPROVED** [review]

**Pull request merged**